source/install/docker-setting-up.rst

```diff
@@ -106,6 +106,8 @@ This command does the following:
 * |opt.pmm-server.latest| is the name and version tag of the image
   to derive the container from.
 
+* A warning message is printed if invalid environment variables are passed in via the command line option ``-e <VARIABLE>``.
```

Comment:
Usually env variables are passed in format `-e <key>=<value>`.
It shows warning message if environment variable key is unknown and fails if environment variable value is invalid.